src/classes/Callable_API.cls

```diff
@@ -59,6 +59,10 @@ global with sharing class Callable_API implements System.Callable {
             } when 'opp.mapstagetostate' {
                 return OPP_StageMappingUtil.determineOppStateFromOppStage((String) params.get('OpportunityStage'));
 
+            } when 'crlp.getrollupdefsforrolluptype' {
```

Comment:
Can we sync in external callable API call to the internal naming, ie use `crlp.getrollupdefinitions` instead of `crlp.getrollupdefsforrolluptype`?

Reply:
I like that it specifies what it getting the definitions for, but `rollups` is repeated in there twice. What do you think about `crlp.getrollupdefinitionsfortype` or `crlp.getdefinitionsforrolluptype`?

Reply:
Either new naming works fine. Good point on keeping the rollup type in the action name. This is also a good example when external action naming differs from internal method name and both are descriptive for their usage.